File: core/src/main/java/com/ustadmobile/core/view/Login2View.java
package com.ustadmobile.core.view;

public interface Login2View extends UstadView{

    String VIEW_NAME = "Login2";
    String ARG_LOGIN_USERNAME = "LoginUsername";

    void setInProgress(boolean inProgress);

    void setErrorMessage(String errorMessage);

    void setServerUrl(String serverUrl);

    void setPassword(String password);

    void forceSync();

    void updateLastActive();

    void updateUsername(String username);

    void setFinishAfficinityOnView();

}

File: core/src/main/java/com/ustadmobile/core/view/Login2View.kt
package com.ustadmobile.core.view

interface Login2View : UstadView {

    fun setInProgress(inProgress: Boolean)

    fun setErrorMessage(errorMessage: String)

    fun setServerUrl(serverUrl: String)

    fun setUsername(username: String)

    fun setPassword(password: String)

    companion object {

        const val VIEW_NAME = "Login2"
    }

}
